frontend/controller/app/chatroom.js

```diff
@@ -41,5 +41,12 @@ sbp('okTurtles.events/on', JOINED_CHATROOM, ({ identityContractID, groupContract
 })
 sbp('okTurtles.events/on', LEFT_CHATROOM, switchCurrentChatRoomHandler)
 sbp('okTurtles.events/on', DELETED_CHATROOM, switchCurrentChatRoomHandler)
+sbp('okTurtles.events/on', NEW_CHATROOM_UNREAD_POSITION, ({ chatRoomID, messageHash }) => {
+  if (messageHash) {
+    sbp('state/vuex/commit', 'setChatRoomScrollPosition', { chatRoomID, messageHash })
+  } else {
+    sbp('state/vuex/commit', 'deleteChatRoomScrollPosition', { chatRoomID })
+  }
+})
```

Comment:
Same RFC - to say where the events are coming from. (If they're coming from different sources, there should be a comment for each source)